server.js

```diff
@@ -123,6 +123,10 @@ if ( cluster.isMaster ) {
   require( './app/api/index' )( app, express, '/api', plugins )
   require( './app/api/index' )( app, express, '/api/v1', plugins )
 
+
+  // init email transport
+  require( './app/email/index' )
```

Comment:
comment out so it doesn't bork out if no .env config is provided for this route, just to make sure...